### DISCUSSION TITLE
随机二次元图片

### DISCUSSION THREAD

**Discussion by fxlqwq** [share121/inter-knot#2036] [其他]
# Github有限制无法随机
## 好看吗好看扣1
电脑：
# http://tk.fxl.us.kg/d
![电脑](http://tk.fxl.us.kg/d)
![电脑](http://tk.fxl.us.kg/d)
手机：
# http://tk.fxl.us.kg/s
![手机](http://tk.fxl.us.kg/s)

**Comment by capacitor1** [comment]
Non-Image content-type returned🤔

**Reply by fxlqwq** [reply]
现在ok了

**Comment by 1luik** [comment]
连接到 tk.fxl.us.kg 时发生错误。SSL 接收到一个超出最大准许长度的记录。

错误代码：SSL_ERROR_RX_RECORD_TOO_LONG

我连不上这玩意

**Comment by ling333920** [comment]
1
